### DISCUSSION TITLE
Can I built a home hub with homespan?

### DISCUSSION THREAD

**Discussion by TIMESTICKING** [HomeSpan/HomeSpan#480] [Q&A] [answered]
At first, I just want to control smart furniture out of my home (or local network), of course, Ipad can do it. But as deeper as I learn, I know a lot about home assistant and home bridge. Speaking of which, is the bridge of homespan a home bridge?

Back to it, Ipad goes with me, Is there any way that allows me to control my furniture while I'm not at home.

**Comment by HomeSpan** [comment, marked as answer]
To control HomeKit accessories from outside your WiFi network using Apple HomeKit you need to have a Home Hub, which is either an Apple TV (generation 4) or an Apple HomePod.  Until recently an Apple iPad kept at home could serve as a Home Hub but, Apple no longer supports this in the latest versions of HomeKit.

HomeSpan cannot be used as a Home Hub.  Note that HomeKit "bridges" are different from a Home Home.  A bridge is just a type of configuration for how a HomeKit Accessory (whether using HomeSpan or a commercial product) operates.
